webhooks/tasks.py

```diff
@@ -0,0 +1,71 @@
+"""
+Celery tasks.
+"""
+import hashlib
+import hmac
+import json
+import logging
+
+from django.apps import apps
+from django.utils.encoding import force_bytes
+import requests
+
+from ccxcon.celery import async
+from webhooks.models import Webhook
+
+log = logging.getLogger(__name__)
+
+
+@async.task
+def publish_webhook(model_str, pk):
+    """
+    Publishes model updates to various services as defined by Webhook instances.
+    """
+    try:
+        app, model_label = model_str.split('.')
+    except ValueError:
+        log.warning("Malformed model string for webhook serialization: %s",
```

Comment:
If it doesn't matter either way why not just log the warning and reraise the exception? In testing we can use `CELERY_ALWAYS_EAGER` to aid in diagnosing these problems, and in production we could turn on `CELERY_SEND_TASK_ERROR_EMAILS`.